x-pack/plugins/siem/public/components/url_state/index.tsx

```diff
@@ -57,6 +57,7 @@ type KeyUrlState = keyof UrlState;
 
 interface UrlStateProps {
   indexPattern: StaticIndexPattern;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Good news -- I think this one is getting cleaned up here: https://github.com/elastic/kibana/pull/36954/files#diff-8fd82466ad8233a23f8deedd497d9058R58

Reply:
yep